lib/main.coffee

```diff
@@ -0,0 +1,60 @@
+{CompositeDisposable} = require 'atom'
+helpers = require 'atom-linter'
+path = require 'path'
+
+module.exports =
+  config:
+    executable:
+      type: 'string'
+      default: 'pylint'
+    rcFile:
```

Comment:
This was part of keeping feature and behavior parity with the current version.  In general it seems ok to expose often used cli flags via the config.  I'm also neutral though :grin: